mock/py/mockbuild/package_manager.py

```diff
@@ -136,15 +145,44 @@ def __init__(self, config, buildroot, plugins, bootstrap_buildroot, is_bootstrap
         self.is_bootstrap_image = is_bootstrap_image
         self.pkg_manager_config = ""
 
+        self.command = config.get(f"{self.name}_command")
+        self.install_command = config.get(f"{self.name}_install_command")
+        self.builddep_command = [self.command, 'builddep']
+        builddep_override = config.get(f"{self.name}_builddep_command")
+        if builddep_override:
+            self.builddep_command = [builddep_override, "builddep"]
+
+        self.common_opts = copy.copy(config.get(f'{self.name}_common_opts', []))
```

Comment:
I am slightly -1 to this. While this is common for all classes, it is slightly hard to read and I would rather keep all these

        self.command = config['dnf_command']
        self.install_command = config['dnf_install_command']

in every class.